packages/redux-subspace/test/enhancers/hierarchyEnhancer-spec.js

```diff
@@ -6,7 +6,7 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-import rootStoreEnhancer from '../../src/enhancers/rootStoreEnhancer'
+import rootStoreEnhancer from '../../src/enhancers/hierarchyEnhancer'
```

Comment:
should probably rename in variable to `hierarchyEnhancer` as well to save someone confusion down the track.